testing-modules/junit-5/src/main/java/groupedvsmultipleassertions/User.java

```diff
@@ -0,0 +1,37 @@
+package groupedvsmultipleassertions;
+
+public class User {
+    String username;
+    String email;
+    Boolean activated;
```

Comment:
These fields should be `private` and I'd suggest `boolean` rather than `Boolean`